wp-content/themes/bh/scss/_responsive.scss

```diff
@@ -3363,9 +3423,14 @@
 					height: 200px;
 				}
 			}
+
+			h2.event-name {

```

Comment:
Structure should be follow exactly the same structure of event.scss, meaning h2.event-name should be declared, in this case, right before .col-2